src/main/java/com/google/api/generator/spring/composer/SpringComposer.java

```diff
@@ -81,7 +81,7 @@ protected static List<GapicClass> addExtraClassAnnotations(List<GapicClass> gapi
     AnnotationNode generatedAnnotation =
         AnnotationNode.builder()
             .setType(FIXED_TYPESTORE.get("Generated"))
-            .setDescription("by gapic-generator-java")
+            .setDescription("by spring-generator-java")
```

Comment:
What do we want to call the spring auto config generator once we move the code to `spring-cloud-gcp`? Maybe we should come up with a long term name and stick to it, `spring-generator-java` seems a little odd to me. Maybe `spring-auto-config-generator` or just a generic `spring-generator`?

Reply:
Thanks for the suggestions. `spring-generator` sounds better to me, to keep it open to expanding beyond auto-config in future.

Reply:
Should we add "google" somewhere in there? For instance, `google-cloud-spring-generator`?